models/campaign.go

```diff
@@ -471,6 +471,44 @@ func DeleteCampaign(id int64) error {
 	return err
 }
 
+//DeleteCampaignByUserId deletes campaigns with a given user id
+func DeleteCampaignByUserId(uid int64) error {
```

Comment:
Minor nitpick - can we make these function names plural? We're deleting multiple campaigns, groups, etc. So if we could name it DeleteCampaignsByUserId, DeleteGroupsByUserId, etc.